src/components/Button/Button.module.css

```diff
@@ -17,6 +17,7 @@
  * 1) Class is doubled to increase specificity
  */
 .button--primary.button--primary {
+  /* 1 */
```

Comment:
I don't think we need any comments inside this block because the `* 1)` comment is describing the class name format (`.button--primary.button--primary`), not the styles inside.